.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "plugins": [
+    "es5"
```

Comment:
👍 maybe add something to the read me or a comment in this file to say why it's es5